backend/.env.example

```
@@ -1,63 +1,76 @@
-# Django
-SECRET_KEY=your-secret-key-here
-DEBUG=True
-ALLOWED_HOSTS=localhost,127.0.0.1
-
-# Database
-DATABASE_URL=sqlite:///db.sqlite3
-
-# JWT
-JWT_SECRET=your-jwt-secret-key-here
-JWT_REFRESH_SECRET=your-jwt-refresh-secret-here
 JWT_ACCESS_TOKEN_LIFETIME=15
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_ACCESS_TOKEN_LIFETIME key should go before the REFRESH_TOKEN_LIFETIME_DAYS key

(UnorderedKey)